scripts/create-all-topics.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+kafka_ready() {
+    kafka-topics.sh --bootstrap-server localhost:9092 --list > /dev/null 2>&1
+}
+
+# Wait for Kafka to be ready
+while ! kafka_ready; do
+    echo "Waiting for Kafka to be ready..."
+    sleep 5
+done
+
+echo "Creating topics now"
+
+kafka-topics.sh --create --bootstrap-server localhost:9092 --topic channel --if-not-exists
+kafka-topics.sh --create --bootstrap-server localhost:9092 --topic language --if-not-exists
+kafka-topics.sh --create --bootstrap-server localhost:9092 --topic flow --if-not-exists
+kafka-topics.sh --create --bootstrap-server localhost:9092 --topic retriever --if-not-exists
```

Comment:
In a separate PR, please update docker-compose.yml to take `retriever` and `indexer` topics.

Also change the retriever and indexer services as well